akka-docs/src/main/paradox/typed/actor-discovery.md

```diff
@@ -30,7 +30,7 @@ applicable.
 When an actor needs to be discovered by another actor but you are unable to put a reference to it in an incoming message,
 you can use the `Receptionist`. You register the specific actors that should be discoverable 
 from other nodes in the local `Receptionist` instance. The API of the receptionist is also based on actor messages. 
-This registry of actor references is then automatically distributed to all other nodes in the cluster. 
+It can be used in an application with a single node configured with a `local` actor provider or in a clustered one configured as `cluster`. This registry of actor references is then automatically distributed to all other nodes in the cluster. 
```

Comment:
Isn't this covered by the fact that the next section is "Cluster Receptionist"?

Reply:
you may be right, @johanandren . In my case, I had to think a bit until I got my head around it. As I see it there's no place where states that those are the only two possible values and think the local provider is never mentioned. I also tried to use remote which doesn't make sense after one understands that there's no way to point to another node. I thought would be useful to state something like above to make explicit what you can deduce from the docs. In case you agree with my suggestion feel free to edit the text. It was just the gist I wanted to present.

Reply:
Perhaps crosslink from here down to the next session with a "The receptionist supports both local and [cluster](#...)" or something if we expect that users will not read on downwards.

Regardless I don't think we should mention the provider, but talk about local vs clustered Akka here, that the provider is what makes a node clustered or local belongs elsewhere IMO (reference conf or cluster section for example).

Reply:
Exactly. I knew you will come up with something cleaner 👍 .